scipy/stats/_distr_params.py

```diff
@@ -10,7 +10,7 @@
     ['beta', (2.3098496451481823, 0.62687954300963677)],
     ['betaprime', (5, 6)],
     ['bradford', (0.29891359763170633,)],
-    ['burr', (10.5, 4.3)],
+    ['burr', (10.0, 5.0)],
```

Comment:
Why change the default parameters?  Do they need to be integers with an integer difference?

Reply:
I did this to make it easier for me to verify via hand calculations, e.g. I can do `\Gamma(9)` but I have difficulty caculating `\Gamma(9.5)` by pen and paper. I can change them back if preferred.